src/library/scala/collection/mutable/AnyRefMap.scala

```diff
@@ -129,7 +129,7 @@ class AnyRefMap[K <: AnyRef, V] private[collection] (defaultEntry: K => V, initi
     e | MissingBit
   }
 
-  private def seekEntryOrOpen(h: Int, k: AnyRef): Int = {
+  @`inline` private def seekEntryOrOpen(h: Int, k: AnyRef): Int = {
```

Comment:
maybe `@inline` is no longer ideal here

Reply:
As shown in the benchmarks above the original `seekEntryOrOpen` was still a bit faster with `@inline` so I left it in.

Reply:
OK, thanks!